modules/serverless-logic/pages/cloud/operator/upgrade-serverless-operator/upgrade_1_34_0_to_1_35_0.adoc

```diff
@@ -0,0 +1,190 @@
+:keywords: kogito, sonataflow, workflow, serverless, operator, kubernetes, minikube, openshift, containers
+// links
+
+:openshift_operator_install_url: https://docs.openshift.com/container-platform/4.13/operators/admin/olm-adding-operators-to-cluster.html
+:openshift_operator_uninstall_url: https://docs.openshift.com/container-platform/4.13/operators/admin/olm-deleting-operators-from-cluster.html
+:kubernetes_operator_install_url: https://operatorhub.io/how-to-install-an-operator
+:kubernetes_operator_uninstall_url: https://olm.operatorframework.io/docs/tasks/uninstall-operator/
+:operatorhub_url: https://operatorhub.io/
+
+This guide describes how to upgrade the {operator_name} 1.34.0 installed in an OpenShift cluster to the version 1.35.0.
+
+.Prerequisites
+* An OpenShift cluster with admin privileges and `oc` installed.
+
+
+
+== Procedure
+
+To upgrade an OSL 1.34.0 installation to the OSL 1.35.0 version, you must execute this procedure:
+
+=== Overall upgrade procedure
+
+We recommend that you read all the steps and understand the procedure before executing.
+Interested parties might automate the procedure according to their convenience or infrastructure, for example, a particular user might have all the SonataFlow CRDs in a github repository, which might help considerably to implement an automation, etc.
+
+. Execute the step `1` of the upgrade for every workflow with the <<workflows_dev_profile, dev profile>>.
+. Execute the steps `1`, `2` and `3` of the upgrade for every workflow with the <<workflows_preview_profile, preview profile>>.
+. Execute the step `1` of the upgrade for every workflow with the <<workflows_gitops_profile, gitops profile>>.
+. Execute the step `1` of the <<data_index_upgrade, Data Index>> upgrade.
+. Execute the step `1` of the <<jobs_service_upgrade, Jobs Service>> upgrade.
+. Upgrade the OSL operator to the version 1.35.0 using the OpenShift management console, and wait until the new version is running, <<check_if_operator_is_updated, see>>.
+. Finalize the <<data_index_upgrade, Data Index>> upgrade by continuing from step `2`.
+. Finalize the <<jobs_service_upgrade, Jobs Service>> upgrade by continuing from step `2`.
+. Finalize the upgrade for the workflows with the <<workflows_gitops_profile, gitops profile>> by continuing from step `2`.
+. Finalize the upgrade for the workflows with the <<workflows_preview_profile, preview profile>> by continuing from step `4`.
+. Finalize the upgrade for the workflows with the <<workflows_dev_profile, dev profile>> by continuing from step `2`.
+
+[#workflows_dev_profile]
+==== Workflows with the `dev` profile
+
+Every workflow with the `dev` profile must be deleted before applying the operator upgrade to OSL 1.35.0, and re-deployed after the upgrade is completed.
+
+For every workflow `my-workflow` with the `dev` profile you must:
+
+*Pre-operator upgrade steps:*
+
+. oc delete -f <my-workflow.yaml>  -n  <target_namespace>
```

Comment:
Instead of directly adding the command as a step you can mention something like this: 

```
Delete the workflow by using the following command: 
[source,terminal]
----
$ oc delete -f <my-workflow.yaml> -n <target_namespace>
----
```

Reply:
added!